stdlib/bigarray.mli

```diff
@@ -1050,3 +1050,47 @@ val reshape_3 :
   ('a, 'b, 'c) Genarray.t -> int -> int -> int -> ('a, 'b, 'c) Array3.t
 (** Specialized version of {!Bigarray.reshape} for reshaping to
    three-dimensional arrays. *)
+
+(** {1:bigarray_concurrency Bigarrays and concurrency safety}
+
+    Reading and writing bigarrays is safe in OCaml, even in presence of data
+    races.
+
+    Unfortunately, C does not offer such guaranteed and any data races anywhere
+    in a C program make the whole program unsafe.
+    Consequently, if bigarrays are used on the C side through the OCaml FFI,
+    data races must be avoided at any cost.
+
+    However, even within OCaml code, it is advised to avoid data races when
+    possible. Data races not only make execution non-deterministic, they also
+    considerably complicate the possible behaviors of programs.
+
+    A general caveat in presence of OCaml data races, concurrent array
```

Comment:
"A general caveat in presence of OCaml data races," -> replace comma with "is that"